apkid/apkid.py

```diff
@@ -97,13 +99,15 @@ def __init__(self, timeout: int = 10, verbose: bool = False, json: bool = False,
         self.typing = typing
         self.entry_max_scan_size = entry_max_scan_size
         self.scan_depth = scan_depth
+        self.include_trackers = include_trackers
```

Comment:
maybe `self.include_trackers` is un-necessary, as not used anywhere else in the file? kept for feedback